blaze-server/src/main/scala/org/http4s/blaze/server/BlazeServerBuilder.scala

```diff
@@ -194,15 +194,37 @@ class BlazeServerBuilder[F[_]] private (
   ): Self =
     copy(sslConfig = new ContextWithClientAuth[F](sslContext, clientAuth))
 
+  /** Configures the server with TLS, using the provided `SSLContext` and its
+    * default `SSLParameters`
+    *
+    * @param sslErrorHandler function that runs if an error occurs during the TLS handshake. Default behavior is to log the error.
+    */
+  def withSslContext(
+      sslContext: SSLContext,
+      sslErrorHandler: PartialFunction[Throwable, Unit] = PartialFunction.empty,
```

Comment:
Wouldn't it be more beneficial to have a general error handler exposed to users and not only for SSL exceptions?

Reply:
IIUC, Ember exposes a general error handler? https://github.com/http4s/http4s/blob/series/0.23/ember-server/shared/src/main/scala/org/http4s/ember/server/EmberServerBuilder.scala#L46-L47

Reply:
For what it's worth, Ember's general error handler just came up in https://github.com/http4s/http4s/pull/7399.  This is more in the direction that Ember is going, but probably away from the direction I'd prefer to go.